app/views/approval_summary_list.scala.html

```diff
@@ -25,11 +25,11 @@
 
```

Comment:
Not your code, but unused imports in this view

Reply:
I've removed the routes import and used "controllers.routes..." in the template (which seems quite common on this template). I've left the AppConfig import as the configuration property gtmContainer is used in the iframe definition in scripts.